core/src/main/scala/com/vladkopanev/zio/saga/Saga.scala

```diff
@@ -142,6 +158,12 @@ object Saga {
   def collectAllPar[R, E, A](sagas: Iterable[Saga[R, E, A]]): Saga[R, E, List[A]] =
     foreachPar[R, E, Saga[R, E, A], A](sagas)(identity)
 
+  /**
+   * Runs all Sagas in iterable in sequence and collects the results.
+   */
+  def collectAll[R, E, A](sagas: Iterable[Saga[R, E, A]]): Saga[R, E, List[A]] =
+    foreach[R, E, Saga[R, E, A], A](sagas)(identity)
```

Comment:
```suggestion
    foreach(sagas)(identity)
```
Should compile without specifying concrete types